src/applications/vaos/new-appointment/components/VAFacilityPage/FacilitiesRadioWidget.jsx

```diff
@@ -47,19 +57,25 @@ export default function FacilitiesRadioWidget({
         ) {
           distance = legacyVAR?.distanceFromCurrentLocation;
         }
+        const facilityPosition = facilityIndex + 1;
 
         return (
-          <div className="form-radio-buttons" key={option.value}>
+          <div
+            className="form-radio-buttons"
+            key={facility.value}
+            aria-atomic="true"
+            aria-live="assertive"
+          >
             <input
               type="radio"
               checked={checked}
-              id={`${id}_${i}`}
+              id={`facility_${facilityPosition}`}
```

Comment:
Same question as the previous page here, too, I'd like to stick with the `id` prop instead of facility in the first part of this string, if possible.

Reply:
replaced id={`facility_${facilityPosition}`} with id={`${id}_${facilityPosition}`}